schemas/app/UOMConversionFactor.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "UOMConversionFactor",
+  "label": "UOM Conversion Factor",
+  "isChild": true,
+  "fields": [
+    {
+      "fieldname": "uom",
+      "label": "UOM",
+      "fieldtype": "Link",
+      "target": "UOM",
+      "create": true,
+      "required": true
+    },
+    {
+      "fieldname": "value",
```

Comment:
This should be called `conversionFactor` or something else that indicates its purpose, not `value`